README.md

```diff
@@ -19,26 +20,48 @@ Please check out our [Contributor's guide](./CONTRIBUTORS.md) on how you can act
 
 # Credits and Special Thanks
 
-Full credits can be found in-game, or wherever the credits.json file is.
+The full credits can be found in-game, or wherever the credits.json file is.

```

Comment:
`The full` can be reverted back to `Full`, and here's the link to `credits.json` as follows:
```suggestion
Full credits can be found in-game, or [in the `credits.json` file](https://github.com/FunkinCrew/funkin.assets/blob/main/exclude/data/credits.json).
```